Added config for printing to console

ElPumpo opened the pull request:
- The 'Sending player ELS data' prints are no longer printed on default, and has been moved to a configuration option.
- Minor cleanup to the resource manifest, having the manifest version at the very top and added the `version` variable.

MrDaGree commented:
I'll merge the resource manifest but not the rest as I don't see the point of you adding the rest?

ElPumpo commented:
Who cares if a player is requesting els info? Thats why i moved it to be a boolean in the config that decides if you want to enable it.

MrDaGree commented:
The reason it matters is mostly for debug purposes when people come asking for support. If everything loads correctly and gets sent correctly then it eliminates a lot of other variables.

ElPumpo commented:
Then tell them to enable debugging? Or at least have debuging enabled by default? People who dont have an issue with this script dont need the useless prints

ElPumpo commented:
I always test my changes before taking stuff online. The change to `__resource.lua` is because external resources can read the manifest information, and get the version. ([example](https://github.com/ESX-PUBLIC/ESX-Snippet/blob/master/general/POC_ESX_UPDATE.lua#L63))

I'll add extra checks for `printDebugInformation` to make it work with older config files and set it to `true` by default.

ElPumpo commented:
There we go, I've changed the code
- if the variable isn't there then it will print stuff like crazy. No just kidding it will print the.. stuff.
- changed varable default to `true`

MrDaGree commented on server/server.lua:
```diff
@@ -370,8 +370,10 @@ function parseVehData(xml, fileName)
-	if outputLoading and outputLoading ~= nil and printDebugInformation then
-		print("Done with vehicle: " .. fileName)
+	if outputLoading and outputLoading ~= nil then
+		if printDebugInformation == nil or printDebugInformation == true then
```
checking if `== true` is pointless for the future. I will still merge this though. Thanks for the contribution!
